docs/releasenotes/6.1.0.rst

```diff
@@ -62,7 +62,7 @@ file. ``ImageFont.FreeTypeFont`` has four new methods,
 :py:meth:`PIL.ImageFont.FreeTypeFont.set_variation_by_name` for using named styles, and
 :py:meth:`PIL.ImageFont.FreeTypeFont.get_variation_axes` and
 :py:meth:`PIL.ImageFont.FreeTypeFont.set_variation_by_axes` for using font axes
-instead. An ``IOError`` will be raised if the font is not a variation font. FreeType
+instead. An ``OSError`` will be raised if the font is not a variation font. FreeType
```

Comment:
Feel free to disagree with me, but at the time of 4.2.0 and 6.1.0, they were IOErrors, so for the sake of anyone using those versions, I would suggest leaving this?

Reply:
Good idea, I've rebased it out.

Reply:
Thanks, but...